configs/projects/puppet-agent.rb

```diff
@@ -129,10 +129,11 @@
   # Vendored modules
   proj.component "module-puppetlabs-augeas_core"
   proj.component "module-puppetlabs-host_core"
-  proj.component "module-puppetlabs-zfs_core"
-  proj.component "module-puppetlabs-zone_core"
   proj.component "module-puppetlabs-selinux_core"
+  proj.component "module-puppetlabs-sshkeys_core"
   proj.component "module-puppetlabs-yumrepo_core"
+  proj.component "module-puppetlabs-zfs_core"
+  proj.component "module-puppetlabs-zone_core"
```

Comment:
Was it annoying you that they weren't alphabetical?

Reply:
A bit. 😆